docs/DatabaseCatalogAPI.yaml

```diff
@@ -367,10 +367,62 @@ components:
           properties:
             locations:
               $ref: "#/components/schemas/Locations"
-        - type: object
-          properties:
             latest_dataset:
               $ref: "#/components/schemas/LatestDataset"
+            gtfs_rt_feeds:
+              type: array
+              items:
+                $ref: "#/components/schemas/BasicGtfsRTFeed"
+
+    BasicGtfsRTFeed:
```

Comment:
Why not use BasicFeed as a "parent"?

Reply:
I was thinking of adding BasicFeed, but BasicFeed has a lot of information that might not be necessary and can be retrieved if the consumer needs it. I'm happy to review this. What do you think?

Reply:
I forgot to mention that BasicFeed doesn't contain `entity_types`.

Reply:
Fine. GtfsRTFeed does contain the entity_type though.
In any case we can live with the duplications for now.